app/components/variable-height-textarea.js

```diff
@@ -47,6 +48,7 @@ export default Ember.TextArea.extend({
       .height(textarea.scrollHeight - 15);
     // scroll to bottom if message typed
     if (_this.get('value') !== "") {
+      Ember.$('.message_textbar').parent().removeClass('has-error');
```

Comment:
Follow `hyphenated words` for class name

Reply:
sure @Darshan-Chauhan , i'll use hyphenated class name.

Reply:
@Darshan-Chauhan, done please re-review.